README copy.md

```diff
@@ -0,0 +1,131 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microcerse] <a name="about-project"></a>
+
+**[Leadersboard]**
+
+> "Leadersboard" is a tool that helps to organize Scores. It simply lists the scores that students has score in a test. In this particular repo, We are building this Leaderboard with using HTML, CSS, JavaScript (ES6), and Webpack.
+
+## 🛠 Built With <a name="built-with"></a>
+- [x] HTML, CSS and JavaScript
+- [x] Linters: Lighthouse, Webhint, Stylelint and ESLint
+- [x] VS Code
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [x] A web browser like Google Chrome
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+### Setup
+
+Clone this repository to your desired folder using the link given below:
+
+use the git clone command to clone this repository or download ZIP
+
+### Install
+
+Install this project with:
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/Odongo006/Leadersboard.git
+```
+### Usage
+
+To run the project, execute the following command: 
+- Repository[@repo](https://github.com/Odongo006/Leadersboard.git)
+
+
+
+
+
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+### Key Features
+- Webpack
+- Linters
+
+### Future Features
+
+- Babel
+- React
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Henry**
+
+- GitHub: [@Odongo006](https://github.com/Odongo006)
+- Twitter: [@HenryOdongo007](https://twitter.com/HenryOdongo007)
+- LinkedIn: [@HenryOdongo](https://www.linkedin.com/in/henry-odongo-91b830182/)
+- Website: [@Portfolio](https://odongo006.github.io/My-Portfolio-Project/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing "></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Odongo006/issues).
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐️
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Hi @Odongo006 kindly named the README correctly so that changes should be featured in your repo. You have two README files in your project directory. Kindly copy the content of **README copy.md** file into **README.md** file and delete **README copy.md** file